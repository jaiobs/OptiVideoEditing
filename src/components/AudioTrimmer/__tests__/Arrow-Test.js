import React from 'react';
import renderer from 'react-test-renderer';
import {Left, Right} from '../Arrow';

describe("Arrow Tests",() => {
    it("should render Left correctly", () => {
        const tree = renderer
        .create(<Left/>)
        .toJSON();
      expect(tree).toMatchSnapshot();
    })

    it("should render Right correctly", () => {
      const tree = renderer
      .create(<Right/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
})