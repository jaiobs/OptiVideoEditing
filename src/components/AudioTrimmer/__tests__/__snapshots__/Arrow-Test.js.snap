// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow Tests should render Left correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "position": "relative",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "height": 10,
          "position": "absolute",
          "width": 2,
        },
        Object {
          "borderTopLeftRadius": 2,
          "borderTopRightRadius": 2,
          "bottom": "49%",
          "transform": Array [
            Object {
              "rotate": "20deg",
            },
          ],
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "height": 10,
          "position": "absolute",
          "width": 2,
        },
        Object {
          "borderBottomLeftRadius": 2,
          "borderBottomRightRadius": 2,
          "top": "49%",
          "transform": Array [
            Object {
              "rotate": "-20deg",
            },
          ],
        },
      ]
    }
  />
</View>
`;

exports[`Arrow Tests should render Right correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "position": "relative",
        "width": "100%",
      },
      Object {
        "transform": Array [
          Object {
            "rotate": "180deg",
          },
        ],
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "height": 10,
          "position": "absolute",
          "width": 2,
        },
        Object {
          "borderTopLeftRadius": 2,
          "borderTopRightRadius": 2,
          "bottom": "49%",
          "transform": Array [
            Object {
              "rotate": "20deg",
            },
          ],
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "height": 10,
          "position": "absolute",
          "width": 2,
        },
        Object {
          "borderBottomLeftRadius": 2,
          "borderBottomRightRadius": 2,
          "top": "49%",
          "transform": Array [
            Object {
              "rotate": "-20deg",
            },
          ],
        },
      ]
    }
  />
</View>
`;
